recipients_app/lib/data/repositories/repositories.dart

```diff
@@ -1,2 +1,6 @@
 export "crash_reporting_repository.dart";
+export "transaction_repository.dart";
 export "user_repository.dart";
+
+const String recipientCollection = "/recipients";
+const String paymentCollection = "payments";
```

Comment:
is this payments or transactions?

Reply:
that's a tricky question, on API / Admin side those are payments (as we are paying recipients some money), 

on our side, we can name them however we want probably, but maybe we can be consistent with the admin / API naming

Reply:
I see. I think it makes it easier for us in the end to be consistent. SInce its also called payments in the UI - should we rename the components?

Reply:
To keep it consistent with the rest of the project naming I've renamed all transactions related stuff to payment [db53fb8](https://github.com/socialincome-san/public/pull/332/commits/db53fb81e3b7d21b624c6bd35be7edb69b832359)